src/Microcks.Testcontainers/Converter/ArrayToStringDeserializer.cs

```diff
@@ -0,0 +1,35 @@
+using System.Text.Json;
```

Comment:
We must add a specific header to remind the License (Apache 2.0) and mention the ownership to be the Microcks Team.